projects/shr-mediator/src/botswana/services/terminology.service.ts

```diff
@@ -148,44 +161,35 @@ export class TerminologyService {
     }
   }
 
-  async getIpmsCode(q: string, c = '') {
+  getIpmsCodeFromMemory(mappings: any[], code: string) {
```

Comment:
Can we enforce typing and avoid "any" ?

This allows to generate automatically typscript from json : https://transform.tools/json-to-typescript

Reply:
Fixed